### ISSUE TITLE
Datasketches support (`quantilesDoubleSketch`)

### ISSUE THREAD

**Issue by jln-pl** [grafadruid/druid-grafana#94]
Hi!
I see that you use a [druid-datasketches](https://github.com/grafadruid/druid-grafana/blob/dc1a65a85c44dbe6a634137661ccc23b9a2a6ec3/docker/druid/environment#L31) extension.

Are you going to add aggregator and post aggregators for [`quantilesDoubleSketch`](https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html)?

**Comment by jbguerraz** [comment]
Hello @jln-pl 
Thank you for this issue!
We would be happy to support that extension, first of all it would need support on the go-druid side of the plugin, mostly like what was done for tdigest sketch (see https://github.com/grafadruid/go-druid/commit/533761d2edab2d9efdf7c1a0d5e0c35abd743775) and then we would need to add the components in this plugin.
my team and I won't have time (and need) for such extension support in a short to mid term, but I'd be happy to help you contribute that feature! that would be awesome! it requires only minimal coding skills, no rocket science :) WDYT ?

**Comment by jln-pl** [comment]
I will try to do it in my spare time ;)

**Comment by jbguerraz** [comment]
I'll be happy to help @jln-pl ! feel free to ping me on Slack for support :)

**Comment by san-san** [comment]
With https://github.com/grafadruid/go-druid/pull/60 merged, Am I correct in saying that we only need changes in the plugin components to support `quantilesDoubleSketch`es?

**Comment by jbguerraz** [comment]
@san-san Yes, we need to add an aggregator component (https://github.com/grafadruid/druid-grafana/tree/master/src/builder/aggregation) and a post aggregation (https://github.com/grafadruid/druid-grafana/tree/master/src/builder/postaggregation) one.

**Comment by san-san** [comment]
Quick update, I have a working solution that we're testing now. We'll raise a PR shortly. https://github.com/san-san/druid-grafana/releases/tag/v.1.3.0_beta if you'd like to try it out.

As a quick note, `quantilesDoublesSketchToQuantiles` has not been implemented in `go-druid`. Once this has been accepted and merged I can work on that.

**Comment by jbguerraz** [comment]
Hello @san-san 
Any chance you moved ahead with it ? need some support ?

**Comment by jbguerraz** [comment]
just in case @san-san if you forgot this but actually made some progress :)
